### ISSUE TITLE
Accessibility issues with the Dropdown in the 

### ISSUE THREAD

**Issue by whmii** [TryGhost/Ghost#9135]
### Issue Summary

https://github.com/TryGhost/Ghost/blob/fbb60adde73e2c5afa1490c2dd1f12c51a95eb9a/core/server/helpers/tpl/navigation.hbs#L3 is incorrectly labeled with the role `role="presentation"` even though it is a navigation link. 

There are `<li class="divider"></li>` items in that list so it is possible that it was meant to apply to these items.

### Steps to Reproduce

1. Open the primary ghost admin dropdown
2. Inspect one of the 6 Navigation items for the attribute `role="presentation"`. *Note:* it has been added to the container `li` element, not the link.

This Issue is also exacerbated by the fact that all links within that attribute have a `tabindex` of `-1` making the entire menu inaccessible via keyboard or screenreader.

I was unable to locate where the `tabindex` was generated. It's likely a component of the overall dropdown behavior.

### Technical details:

* Ghost Version: 1.12.0
* Node Version: N/A
* Browser/OS: N/A
* Database: N/A

**Comment by kevinansfield** [comment]
Hey @whmii 👋 

Definitely agree that the accessibility markup can be improved in a lot of places in Ghost. I think there's a little confusion in your issue because you're referencing the front-end [navigation.hbs](https://github.com/TryGhost/Ghost/blob/master/core/server/helpers/tpl/navigation.hbs) template that is used in Ghost themes but you're talking about the [dropdown in Ghost-Admin](https://github.com/TryGhost/Ghost-Admin/blob/master/app/templates/components/gh-nav-menu.hbs)

I'm opening a PR that improves the markup in the front-end [navigation.hbs](https://github.com/TryGhost/Ghost/blob/master/core/server/helpers/tpl/navigation.hbs) template.

On the Ghost-Admin side, the `role="presentation"` in the `<li>` tags isn't an issue because the `<ul>` and `<a>` elements are marked up with the explicit `menu` and `menuitem` roles. I agree the `tabindex="-1"` probably shouldn't be there though and we could add further improvements by using the correct `aria-haspopup/hidden/expanded` attributes.

**Comment by kevinansfield** [comment]
[Front-end PR](https://github.com/TryGhost/Ghost/pull/9137) has been opened. As mentioned in the PR comment the front-end accessibility markup is intentionally minimal because we don't know where the `{{navigation}}` helper will be used in a theme. Theme developers should override if applicable or ensure they wrap the helper in more suitable navigation markup.
